remixer_core/src/main/java/com/google/android/libraries/remixer/Remixer.java

```diff
@@ -117,7 +120,7 @@ public void addItem(RemixerItem remixerItem) {
     }
     listForKey.add(remixerItem);
     remixerItem.setRemixer(this);
-    remixerItems.add(remixerItem);
+    getRemixerItemsForContext(remixerItem.getContext()).add(remixerItem);
```

Comment:
Not really, though the null context doesn't make sense, the null key is valid as HashMap key.